backend/src/event/Event.ts

```diff
@@ -308,4 +446,5 @@ export class Event {
   transactionId?: number
   contributionId?: number
   amount?: decimal
+  message?: string
```

Comment:
sorry i overlooked this the last time,
- for this new attribute a database migration for the table event_protocol is necessary
  - so please create a database migration script and all typeORM changes to add the new column "message_id: unsigned int(10)
- please rename here the attribute "message" to "messageId" and change the type to number

we don't want to overload the eventprotocoll with business data, identifiers to get the business content later will be sufficient

Reply:
I think I did it right, haven't done it many times before so please, take a look at it if possible, I think it is allright

Thank you beforehand!